src/sys/socket/sockopt.rs

```diff
@@ -7,7 +7,6 @@ use cfg_if::cfg_if;
 use libc::{self, c_int, c_void, socklen_t};
 use std::ffi::{OsStr, OsString};
 use std::mem::{self, MaybeUninit};
-#[cfg(target_family = "unix")]
```

Comment:
Note  Fuchsia may not be considered Unix forever. https://github.com/rust-lang/rust/issues/58590

Reply:
Get it, thanks for showing me this!